### ISSUE TITLE
'left handed' mode for drawing tablets that have integrated touchpads does not work

### ISSUE THREAD

**Issue by elmarsto** [swaywm/sway#6564]
Unsure if this is libinput or sway (I'm new to both) but here goes. If this is for libinput please help me route the issue; it's a dealbreaking blocker.

 - sway v1.6.1
 - Wacom Intuos (Pro, I believe)

Description:

It's possible to set left_handed mode on the pen part of a tablet but not the touch part, resulting in an extraordinarily disorienting user experience. The pen works as expected, but touch gestures are inverted (when the tablet is held with the buttons on the right and the drawing surface on the left.) This means that when you drag your finger left, it goes right, and so forth.

```
    input type:tablet_tool {
          map_to_output "${config.lattice.wacom.monitor}"
          #left_handed enabled
    }
    input type:tablet_pad {
          map_to_output "${config.lattice.wacom.monitor}"
          # Doesn't work
          #left_handed enabled
    }
```

**Comment by MaeIsBad** [comment]
This is an ancient bug at this point, I think it can be closed. I have a wacom intos pro and I'm able to setup left handed mode with

```
swaymsg input 1386:789:Wacom_Intuos_Pro_M_Finger left_handed enabled
```

**Comment by emersion** [comment]
Please re-open if you can still reproduce!

**Comment by MaeIsBad** [comment]
Nevermind, looks like the issue is still a thing. I'm convinced this is a libinput problem, I'll make a repro case and open an upstream bug

**Comment by MaeIsBad** [comment]
Yeah after some debugging I can 100% confirm this is a distro packaging bug.
For the sake of documentation I originally used an ubuntu machine to test, which packages everything correctly and works out of the box.
On my nixos machine things didn't work, I needed to make sure the correct udev rules(libinput and libwacom) were installed. 
This is because libinput relies on the libinput udev rules to group the tablet, pen and finger evdev in a single device group and the libwacom udev rules to actually mark which devices are tablets.
On nix the fix is to add 
```
  services.libinput.enable = true;
  services.udev.packages = [pkgs.libwacom.out];
```
to your configuration
